compose/cli/main.py

```diff
@@ -256,16 +257,43 @@ def ps(self, project, options):
         Usage: ps [options] [SERVICE...]
 
         Options:
-            -q    Only display IDs
+            -q          Only display IDs
+            -a, --all   Display orphan containers
         """
+        orphan = options.get('--all', False)
         containers = sorted(
             project.containers(service_names=options['SERVICE'], stopped=True) +
             project.containers(service_names=options['SERVICE'], one_off=True),
             key=attrgetter('name'))
 
+        orphan_containers = sorted(
+            project.containers(stopped=True, orphan=True) +
+            project.containers(one_off=True, orphan=True),
+            key=attrgetter('name'))
```

Comment:
I did this to keep orphaned containers at the end of the list, I thought it was more clear to not mix up orphaned containers with active services, but I will change this.